automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSpec.scala

```diff
@@ -277,37 +266,49 @@ class LeonardoSpec extends FreeSpec with Matchers with Eventually with ParallelT
     }
 
     "should create, monitor, and delete a cluster" in {
-      implicit val token = ronAuthToken
-      withNewCluster(project) { _ =>
-        // no-op; just verify that it launches
-      }
+      val ownerToken = hermioneAuthToken
```

Comment:
Since all tests now do this I wonder if we can just assume hermione/ron instead of specifiying it in each test.

Reply:
Good idea.  I updated `withNewBillingProject` but I'd prefer to keep `withNewCluster` more flexible.